### ISSUE TITLE
ArchLabs install: nwg-shell-config cannot save settings

### ISSUE THREAD

**Issue by humanplayer2** [nwg-piotr/nwg-shell#12]
On a fresh the ArchLabs install with sway session, nwg-shell-config launches on start, but its settings do not take effect and are not saved.

Running it from terminal and hitting Apply returns:

```
Saving /home/humanplayer2/.local/share/nwg-shell-config/preset-0
Saving /home/humanplayer2/.local/share/nwg-shell-config/preset-1
Saving /home/humanplayer2/.local/share/nwg-shell-config/preset-2
Saving /home/humanplayer2/.local/share/nwg-shell-config/preset-3
Saving /home/humanplayer2/.local/share/nwg-shell-config/custom
Saving swaync settings to /home/rasmus/.config/swaync/config.json
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/nwg_shell_config/main.py", line 289, in on_apply_btn
    update_swaync_config(preset["swaync-positionX"],
  File "/usr/lib/python3.10/site-packages/nwg_shell_config/main.py", line 760, in update_swaync_config
    save_json(swaync_settings, settings_file)
  File "/usr/lib/python3.10/site-packages/nwg_shell_config/tools.py", line 106, in save_json
    with open(path, 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/humanplayer2/.config/swaync/config.json'
```

**Comment by nwg-piotr** [comment]
That's because the updated skel package is still in testing. Hopefully Nate will push it to the stable repo, and release the new iso tonight.

**Comment by humanplayer2** [comment]
Thanks! They have now landed in ArchLabs 2022.05.29 Release, where the issue is solved.
